### PR TITLE
liveinst: Port to polkit

### PR DESCRIPTION
**Pull request by halfline**
liveinst currently uses consolehelper to elevate user privilges to root in preparation for running anaconda.

consolehelper is deprecated and has been for a very long time.

Furthermore, consolehelper can't be used from gnome-initial-setup which we'll need in the near future.

This commit ports from consolehelper to polkit instead.

### CODE REVIEW

**Comment by halfline** [comment]
(i'm not exactly sure how anaconda.spec.in changes work, it seems like the file might be partly generated?)

File: data/liveinst/liveinst [hunk 1]
```diff
@@ -21,6 +21,12 @@
 # Allow running another command in the place of anaconda, but in this same
 # environment.  This allows storage testing to make use of all the module
 # loading and lvm control in this file, too.
+if [ "$(id -u)" -ne 0 ]; then
+    xhost +si:localuser:root
+    unset XAUTHORITY
+    exec pkexec "$0" "$@"
+fi
+
 ANACONDA="${LIVECMD:=anaconda --liveinst --graphical}"
```
Comment (VladimirSlavik) [inline data/liveinst/liveinst:30]: I think the comment belongs to the old code, and the new code could be described as "Always run with root privileges"...?

**Review by VladimirSlavik — APPROVED** [review]
Thank you! Spec and make looks good to me.

Did you test this somehow, or do we need to do it?

File: data/liveinst/liveinst (hunk 1 above)
Comment (halfline) [reply data/liveinst/liveinst]: indeed. I pushed some comment changes. let me know if it looks better to you

**Comment by VladimirSlavik** [comment]
FYI: Unit tests can't succeed because of GPG keys mismatch after recent fedora branching. Otherwise they would, this is not tested at all.

**Comment by M4rtinK** [comment]
So looks like this works at least as far as Anaconda starting successfully to the Web UI. :) One thing I've noticed, is that it shows some authentication dialog now:
live_image_auth_question
That's not the best UX (and kinda irrelevant) but the UI shows up in a coupler seconds once the dialog is accepted.

Now trying to have an installation run to the end to see if that works as well (including shutting down Anaconda and the environment cleanly). The test image I'm using is here:

https://fedorapeople.org/groups/anaconda/webui/temp/Fedora-Workstation_test_ray_liveinst_changes.iso

**Review by M4rtinK — APPROVED** [review]
So I'm able to start the installer & the installation seems to result in a working system, so looks good to me. :)

**Review by jkonecny12 — APPROVED** [review]
Looks great to me too. Thanks a lot for improving our outdated script!

**Review by vojtechtrefny — APPROVED** [review]

**Pull request merged**